Lib/test/test_regrtest.py

```diff
@@ -821,5 +821,19 @@ def test_crashed(self):
                                   randomize=True)
 
 
+class TempCwdTestCase(unittest.TestCase):
```

Comment:
Please move this test to test_support.py which tests test.support.

test_regrtest tests Lib/test/libregrtest/.